extensions/raycast-nrm/CHANGELOG.md

```diff
@@ -0,0 +1,5 @@
+# raycast-nrm Changelog
+
+## [1.0.0] - 2023-11-18
+- 支持静态地址源获取
+- 支持npm地址源切换
```

Comment:
At the moment, Raycast doesn't support localization and only supports US English. Therefore, please avoid introducing your custom way to localize your extension.